war/src/main/js/add-item.js

```diff
@@ -246,15 +257,22 @@ $.when(getItems()).done(function(data) {
             showInputHelp('.add-item-name');
             setFieldValidationStatus('name', true);
             if (getFormValidationStatus()) {
-              enableSubmit(true);
+              switch (event.type) {
+                case 'blur':
+                  enableSubmit(true);
+                  break;
+                case 'keyup':
+                  enableButton(true);
```

Comment:
@daniel-beck @lanwen Is this how you suggested to separate focusing part of enableSubmit feature?